cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -132,6 +132,11 @@ protected void execute() throws CommandException {
         consoleWriter.fg(Ansi.Color.GREEN).newLine().a("Checks completed").println(2);
     }
 
+    private boolean isPhabricatorSkipChecks() {
```

Comment:
knit: not used?

Reply:
Decided to move this implementation to a different PR and this must've slipped into a later commit on this branch